app/config/routing.yml

```diff
@@ -40,6 +36,15 @@ MgatePersonneBundle:
     resource: "@MgatePersonneBundle/Resources/config/routing.yml"
     prefix:   /personne
 
+N7consulting_rh:
+    resource: "@N7consultingRhBundle/Resources/config/routing.yml"
```

Comment:
you don't need the routing in a dedicated bundle, instead you should add it here IMO